tree/30_generic_methods/schedule_simple_catchup.cf

```diff
@@ -37,7 +37,6 @@
 # @parameter periodicity_minutes   How often should the job run
 # @parameter periodicity_hours     How often should the job run
 # @parameter periodicity_days      How often should the job run
-# @parameter mode                  "nodups": avoid duplicate runs in the same period / "catchup": avoid duplicates and one or more run have been missed, run once before next period / "stateless": no check is done on past runs
```

Comment:
This is a hard incompatibility that may break users policy, but as this method always broke policies when used from the technique editor, it should not be an issue...

Reply:
this parameter should not be there 
And it never worked from the technique editor, so tyhis is ok